code/modules/organs/blood.dm

```diff
@@ -74,12 +74,12 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 
 			if(!heart)
 				blood_volume = 0
-			else if(heart.damage > 1 && heart.damage < heart.min_bruised_damage)
-				blood_volume *= 0.8
-			else if(heart.damage >= heart.min_bruised_damage && heart.damage < heart.min_broken_damage)
-				blood_volume *= 0.6
-			else if(heart.damage >= heart.min_broken_damage && heart.damage < INFINITY)
-				blood_volume *= 0.3
+			else if(heart.is_broken()) //I honestly hope this doesn't fuck up surgery or anything. Might need to test heart surgery on a patient.
```

Comment:
Do that and confirm that there are no issues.

Reply:
Did some tests. The patient survives if their heart is operated on, however if the patient loses so much as 1 particle of blood during the operation, it will instantly kill them because of how bloodcode works.

Having a broken heart is now less fatal in the short run, but still as fatal as it was in the long run.